specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json

```diff
@@ -1155,7 +1333,7 @@
           "$ref": "#/definitions/WorkspaceProperties",
           "description":"Workspace properties." 
         },
-        "eTag": {
+        "etag": {
```

Comment:
Was this case change due to the Linter alone? I was chatting with @lmazuel, and he mentioned that sometimes the linter scares people into doing the wrong thing in this circumstance. IF the service really does return `eTag` instead of `etag` we should add a suppression instead.

Reply:
You are right. I changed to be eTag and added suppression.